app/src/main/res/values/styles.xml

```diff
@@ -6,6 +6,10 @@
         <item name="android:indeterminateTint">@color/black</item>
     </style>
 
+    <style name="FragmentLoadingBarStyle">
+        <item name="android:layout_height">@dimen/loading_bar_text_height</item>
```

Comment:
this property is not necessary because you already have it on the view

Reply:
Done thx